tests/chain_swaps.rs

```diff
@@ -257,6 +248,43 @@ fn bitcoin_liquid_v2_chain() {
     }
 }
 
+fn refund_bitcoin_liquid_v2_chain(
+    lockup_script: BtcSwapScript,
+    refund_address: String,
+    swap_id: String,
+    our_refund_keys: Keypair,
+    boltz_api_v2: BoltzApiClientV2,
+    absolute_fees: u64,
+) {
+    let refund_tx = BtcSwapTx::new_refund(
+        lockup_script.clone(),
+        &refund_address,
+        &ElectrumConfig::default_bitcoin(),
+        BOLTZ_TESTNET_URL_V2.to_owned(),
+        swap_id.clone(),
+    )
+    .unwrap();
+    let tx = refund_tx
+        .sign_refund(
+            &our_refund_keys,
+            Fee::Absolute(absolute_fees),
+            Some(Cooperative {
+                boltz_api: &boltz_api_v2,
+                swap_id: swap_id.clone(),
+                pub_nonce: None,
+                partial_sig: None,
+            }),
+        )
+        .unwrap();
+
+    refund_tx
+        .broadcast(&tx, &ElectrumConfig::default_bitcoin())
+        .unwrap();
+
+    log::info!("Succesfully broadcasted claim tx!");
+    log::debug!("Claim Tx {:?}", tx);
```

Comment:
It is not a claim but a refund transaction

Reply:
Right, didn't notice because it was already wrong :) 

Fix: [de084f0](https://github.com/SatoshiPortal/boltz-rust/pull/87/commits/de084f0c73fe78f8a1421b7278fa3fcf42301640)